pkgs/top-level/all-packages.nix

```diff
@@ -15795,7 +15795,7 @@ with pkgs;
 
   webcamoid = libsForQt5.callPackage ../applications/video/webcamoid { };
 
-  webcord = callPackage ../by-name/we/webcord/package.nix { electron = electron_32; };
+  webcord = callPackage ../by-name/we/webcord/package.nix { electron = electron; };
 
   webcord-vencord = callPackage ../by-name/we/webcord-vencord/package.nix { electron = electron_31; };
```

Comment:
And the actual package flagged and affected by #370758 isn't `webcord`, it's `webcord-vendord`.

`webcord` was already at `electron_32`, which won't be EOL for another two months or so.

`webcord-vendord` on the other hand is still at `electron_31`.